### ISSUE TITLE
Using FailoverClusterDsc for creating domain independant / workgroup Windows Cluster

### ISSUE THREAD

**Issue by mikedavem** [dsccommunity/FailoverClusterDsc#280]
Hi all,

I used serveral times FailoverClusterDsc for provisioning Windows Failover Cluster + availability groups. Thank you to the community :)

I'm currently trying to do the same but for a specific scenario that includes workgroup availability group (https://learn.microsoft.com/en-us/azure/azure-sql/virtual-machines/windows/availability-group-clusterless-workgroup-configure?view=azuresql). In this context, I need to provision a domain independant cluster (https://techcommunity.microsoft.com/t5/core-infrastructure-and-security/domain-independent-cluster-and-availability-group/ba-p/371462) .

Basically something like that (with -AdministrativeAccessPoint DNS) :

````
New-Cluster –Name NoDomainClus -Node Server1,Demo2 -AdministrativeAccessPoint DNS `
-StaticAddress 10.0.0.41 -NoStorage -IgnoreNetwork 192.168.1.0/24
```

According to the FailoverClusterDsc documentation , this scenario cannot be addressed currently. Am I right ? 

Thank you

**Comment by johlju** [comment]
You are right, it is tracked in https://github.com/dsccommunity/FailoverClusterDsc/issues/147. Happy to review a PR that suggest changes to support this.

**Comment by mikedavem** [comment]
Apologize for this late answer. I didn't contributed yet but I did the exercice for a specific customer need with existing SqlServerDsc modules. Because we don't rely anymore to an Active Directory domain, lot of stuff must be modified / added ... here a first feedback:

- Adding host entries for each replica + DNS suffix
- Enabling PSRemoting / CredSSP for remote authentication (with workgroup user) + remote execution of PowerShell commands
- Current Failover DSC module doesn't support AdministrativeAccessPoint mode
- Current SQLEndpoint DSC module doesn't support authentication from certificates (required with workgroup AGs)
